source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -624,6 +626,10 @@ def get(self, region=None, lon=None, lat=None, uri=None):
             args['max_bss_duration_to_pt'] = args['max_duration_to_pt']
             args['max_car_duration_to_pt'] = args['max_duration_to_pt']
 
+        if args['data_freshness'] is None:
+            # retrocompatibilty handling
+            args['data_freshness'] = 'adapted' if args['disruption_active'] is True else 'theoric'
```

Comment:
I personally vote for `scheduled` and 'amended`, but I think it's better to wait for andrew